config/dev.exs

```diff
@@ -14,6 +14,11 @@ config :jumubase, JumubaseWeb.Endpoint,
   watchers: [node: ["node_modules/brunch/bin/brunch", "watch", "--stdin",
                     cd: Path.expand("../assets", __DIR__)]]
 
+# Configure Phauxth authentication
+config :phauxth,
+  token_salt: System.get_env("PHAUXTH_TOKEN_SALT") || "1J9hSjU4",
+  endpoint: JumubaseWeb.Endpoint
```

Comment:
Should we set the endpoint once in `config.exs`? Now that line is in three different files.

Reply:
Good idea! Will change that.